SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -3203,6 +3205,44 @@ \section{Ternary Operator}
 And hence, the ternary operator is accepted in such cases.
 
 
+\section{Backslashes}
+\label{sec:Backslashes}
+
+The use of a backslash as line continuation character appears to sometimes
+hamper code readability. A backslash at the end of a line that has C++ \code{//}
+comment may be quite confusing. For example, it may be overlooked easily that
+the following code has commented out the \code{Sleep()} function call:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
+  // Wil we go to sleep or not? \
+  Sleep();
+\end{minted}
+\normalsize
+
+Moreover, the use of backslashes in a multiline string literal also appears
+troublesome. For example, the following \code{badDescription} has all the space
+characters of the indentation between "developers with " and "an extensive suite"
+embedded inside the string, which may not be intended. The definition of
+\code{goodDescription} shows how to define such a lengthy string properly (as
+the compiler concatenates adjacent string literals automatically).
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
+    const char badDescription[] = "ITK is an open-source, cross-platform library that provides developers with \
+                                   an extensive suite of software tools for image analysis.";
+
+    const char goodDescription[] = "ITK is an open-source, cross-platform library that provides developers with "
+                                   "an extensive suite of software tools for image analysis.";
+\end{minted}
+\normalsize
+
+So in general, the use of a backslash as line continuation character is
+discouraged. Exception: a lengthy preprocessor macro definitions may actually
+be more readable when it spans multiple lines, so in this case, the use of
+backslashes is OK. But even then, such line continuation characters should not
+appear inside a string literal.
```

Comment:
So depending on my above comment, we should never use the backslash. It is not only discouraged: any PR that has one is likely to get a comment on it requesting to be changed.

I would put an example or cross-reference the appropriate section (if it exists, or maybe create one if it doesn't) concerning the marcos. Have not reviewed our definitions, but maybe we use the backslash always? Not sure what the rule has been/a pattern exists.

I comment on the style: I would rewrite the second sentence so that we integrated "Exception" into the sentence and would change "OK" for e.g. "accepted"/", ecommended".

Reply:
Backslashes are routinely used in macros, e.g. [itkSimpleNewMacro](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc04/Modules/Core/Common/include/itkMacro.h#L316-L327).